modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -301,4 +302,132 @@ Set `auto_replay_nacks` to `false` to delete rejected messages. Disabling auto r
 
 *Default*: `true`
 
+=== `oauth`
```

Comment:
Please check these fields descriptions as I have reworded slightly to make them more generic.

Reply:
I think this is fine, but please note these fields are also exposed by other components such as https://docs.redpanda.com/redpanda-connect/components/inputs/http_client/#oauth. I'm not sure what's the best way to ensure such common fields don't end up with varying descriptions across the docs...

Reply:
Raising a ticket to add a content that can be reused and conditioned across different components/connectors: https://github.com/redpanda-data/documentation-private/issues/2760